src/System.Security.Cryptography.X509Certificates/src/Internal/Cryptography/Helpers.cs

```diff
@@ -83,6 +72,14 @@ private static byte HexToByte(char val)
                 return 0xFF;
         }
 
+        private static char ByteToHex(byte b)
```

Comment:
Should it be called NibbleToHex? And why does it return "?" in what seems like an error case?